at_secondary/at_secondary_server/lib/src/notification/stats_notification_service.dart

```diff
@@ -39,70 +39,75 @@ class StatsNotificationService {
   }
 
   final _logger = AtSignLogger('StatsNotificationService');
-  final String _currentAtSign =
-      AtSecondaryServerImpl.getInstance().currentAtSign;
-  AtCommitLog? _atCommitLog;
-  DateTime _lastSentTime = DateTime.now().toUtc();
+  late String currentAtSign;
+  AtCommitLog? atCommitLog;
+  List<InboundConnection>? connectionsList;
```

Comment:
I believe this for the unit tests. Should we keep it the way it was and mock "InboundConnectionPool.getInstance().getConnections();" and return the connections you need for the testing.

Reply:
Tried the approach of mocking InboundConnectionPool. It is tricky to mock InboundConnections.